components/alert/README.md

```diff
@@ -4,21 +4,38 @@ Alerts communicate critical information relating to the state of the system and
 <!-- docs: demo autoSize:false align:start -->
 ```html
 <script type="module">
-	import '@brightspace-ui/core/components/alert/alert-toast.js';
+  import '@brightspace-ui/core/components/alert/alert-toast.js';
+  import '@brightspace-ui/core/components/button/button.js';
+
+  var alert = document.querySelector('#alert');
+  var alertToast = document.querySelector('#alert-toast');
+  var button = document.querySelector('#open');
+
+  alert.addEventListener('d2l-alert-close', function() {
```

Comment:
This code checks if both alerts are closed and if so displays the button. Getting the layout to work with the button always displayed was a bit complex.